pkg/api/defaults-apiserver_test.go

```diff
@@ -536,3 +536,29 @@ func TestAPIServerAnonymousAuth(t *testing.T) {
 			a["--anonymous-auth"])
 	}
 }
+
+func TestAPIServerConfigChangeVerbosity(t *testing.T) {
+	// Test
+	// "apiServerConfig": {
+	// 	"--v": "2"
+	// },
+	cs := CreateMockContainerService("testcluster", defaultTestClusterVer, 3, 2, false)
+	cs.Properties.OrchestratorProfile.KubernetesConfig.APIServerConfig = map[string]string{
+		"--v": "2",
```

Comment:
And if we change the default to 2, let's update this UT case to override to 4 instead